pages/kids/stories/[story].tsx

```diff
@@ -43,7 +43,7 @@ const StoriesContainer = ({ story, phrases }: StoriesProps): ReactNode => {
       <SEO
         title={`${title_current + ' - '}` + t(`seo.kids_page_storiesTitle.${getCountryVariant()}`)}
         description={t(`seo.kids_page_storiesDescription.${getCountryVariant()}`)}
-        image={`https://www.movapp.cz/kids/' + ${image_current_story} + '.jpg'`}
+        image={`'https://www.movapp.cz/kids/' + ${image_current_story} + '.jpg'`}
```

Comment:
I think you don't need any ...
image={`https://www.movapp.cz/kids/${image_current_story}.jpg`}

Reply:
we have a dynamic parameter ${}, we should use ``
or are you talking about a different way of specifying the link?

Reply:
@jejdacz Oh ,yes, you are right. My developing skills are old and obsolete ;)

Reply:
Marek is using `` but github makes a mess with comments formating.

Reply:
Okay. Go it. I will change it tomorrow)